doc/index.rst

```diff
@@ -3,29 +3,48 @@ PyInstaller Manual
 ==================
 
 :Version: |PyInstallerVersion|
-:Homepage: |Homepage|
+:Homepage: `<https://pyinstaller.org/>`_
 :Contact: pyinstaller@googlegroups.com
 :Authors: David Cortesi, based on structure by Giovanni Bajo & William Caban, based on Gordon McMillan's manual
 :Copyright: This document has been placed in the public domain.
 
 
-|PyInstaller| bundles a Python application and all its dependencies into
-a single package.
+PyInstaller bundles a Python application and all its dependencies into a single package.
 The user can run the packaged app without installing a Python interpreter or any modules.
-|PyInstaller| supports Python 3.7 or newer,
-and correctly bundles the major Python packages
+PyInstaller supports Python 3.7 and newer, and correctly bundles many major Python packages
 such as numpy, PyQt, Django, wxPython, and others.
```

Comment:
Suppose we drop `Django` from this list?

Reply:
Good idea. Replaced it with matplotlib.